Source/MetroXHunter/Interaction/Locker.h

```diff
@@ -39,6 +50,9 @@ class METROXHUNTER_API ALocker : public ABaseInteractable
 	UFUNCTION( BlueprintCallable, Category = "Locker|Skill Check" )
 	void EndSkillCheck();
 
+	UFUNCTION( BlueprintCallable, Category = "Locker|Loot" )
+	void SpawnLootItem();
+
 protected:
 	UPROPERTY( BlueprintReadOnly, Category = "Locker|Skill Check" )
```

Comment:
Avoid spaces in the category, Unreal does the spacing for you in the editor

`"Locker|SkillCheck"`* (x2)